pom.xml

```diff
@@ -670,13 +707,30 @@
 		<dependency>
 			<groupId>io.jenkins.plugins</groupId>
 			<artifactId>javax-activation-api</artifactId>
-			<version>1.2.0-3</version> <!-- TODO use version from BOM when possible -->
 		</dependency>
 
 		<dependency>
 			<groupId>io.jenkins.plugins</groupId>
 			<artifactId>jaxb</artifactId>
-			<version>2.3.6-1</version> <!-- TODO use version from BOM when possible -->
+		</dependency>
+		<!-- Explicit JAXB API -->
+		<dependency>
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
+			<version>2.3.2</version>
+		</dependency>
+
+		<!-- Explicit JAXB Runtime -->
+		<dependency>
+			<groupId>org.glassfish.jaxb</groupId>
+			<artifactId>jaxb-runtime</artifactId>
+			<version>2.3.2</version>
+		</dependency>
```

Comment:
Unclear why this is needed.

Reply:
Hi @Basil,
Thank you for review :)
These 2 dependencies solved the JAXBException on junit tests.
However, this PR is just temporary and private for a customer.
For next release we are going to migrate the plugin from JDK1.8 to JDK21, so any suggestions are welcome.

Reply:
Is that because your application needs version 2.3.2, as opposed to the 2.3.1 provided by the Jenkins JAXB plugin? In general it is a bad practice to provide the same classes via two JAR files, and I think the relevant classes should already be on the classpath via the `io.jenkins.plugins:jaxb` plugin (albeit at the older 2.3.1 version). In general, invoking `JAXBContext.newInstance` on Jenkins requires setting the thread's context class loader as in https://github.com/jenkinsci/cppcheck-plugin/pull/64 for the reasons described in [this document](https://www.jenkins.io/doc/developer/plugin-development/dependencies-and-class-loading/).